packages/main/src/components/FilterItem/index.tsx

```diff
@@ -52,6 +54,17 @@ const FilterItem: FC<FilterItemPropTypes> = forwardRef((props: FilterItemPropTyp
     onChange(Event.of(null, e.getOriginalEvent(), { selectedItem: item }));
   }
 
+  function onMultiCbChange(e) {
```

Comment:
would it make sense to use `useCallback` here?

Reply:
Thought about it as well... 
I think it would be a micro optimization because the only place where its used is in filterComponent which gets memoized anyway (the memoization won´t run again if onSelect or onMultiCbChange get a new reference). Therefore there is no unnecessary rerender when the reference of onMultiCbChange and onSelect changes. Thats exactly the goal of useCallback. So almost no difference with or without useCallback.